Docs: Link to search with query rules page from query rules DSL (#98269)
elastic/elasticsearch: docs/reference/query-dsl/rule-query.asciidoc

preview::[]

Applies <<query-rules-apis,query rules>> to the query before returning results.
This feature is used to promote documents in the manner of a <<query-dsl-pinned-query>> based on matching defined rules.
If no matching query rules are defined, the "organic" matches for the query are returned.

[NOTE]
====

> In order to use the rule query, you must have a query ruleset defined with rules to query.
> You can use the <<query-rules-apis, query rules management APIs>> to create and manage query rules.

Reviewer: ```suggestion
To use the rule query, you first need a defined set of query rules.
Use the <<query-rules-apis, query rules management APIs>> to create and manage query rules.
```

Author: I like that, thank you!

      "criteria": [
        {
          "type": "exact",

>           "metadata": "user_query",

Reviewer 2: Nice clarification 👍
